assets/css/custom.css

```
@@ -1896,4 +1905,566 @@ a.read-all-posts:hover {
+  padding: 30px 15px 0;
+  /* padding: 30px 0 0; */
+}
+@media (min-width:881px) {
+  .new-header .container {
+    /* padding: 30px 40px 0; */
+    padding: 30px 0 0;
+  }
+}
+
+@media screen and (max-width: 880px) {
+  .new-header .container {
+    padding-bottom: 22px;
+    padding-top: 22px;
+  }
+}
+
+.new-header .container .logo_link {
+  margin-right: auto;
+  padding-bottom: 26px;
+  transition: .25s;
+  width: 30%;
+}
+
+@media screen and (max-width: 880px) {
+  .new-header .container .logo_link {
+    padding-bottom: 0;
+    width: 22rem;
+  }
+}
+
+.hamburger {
+  background-color: transparent;
+  border: 0;
+  color: inherit;
+  cursor: pointer;
+  display: inline-block;
+  display: none;
+  font: inherit;
+  margin: 0 0 0 1rem;
+  overflow: visible;
+  text-transform: none;
+  transform: scale(.75);
+  transform-origin: center;
+  transition-duration: .15s;
+  transition-property: opacity,filter;
+  transition-timing-function: linear;
+}
+
+@media screen and (max-width: 880px) {
+    .hamburger {
+      display: block;
+  }
+}
+
+.hamburger-box {
+  display: inline-block;
+  height: 2.5rem;
+  position: relative;
+  width: 3.5rem;
+}
+
+.hamburger-inner:after, .hamburger-inner:before {
+  content: "";
+  display: block;
+}
+
+.hamburger-inner {
+  display: block;
+  margin-top: -0.125rem;
+  top: 50%;
+}
+.hamburger-inner, 
+.hamburger-inner:after, 
+.hamburger-inner:before {
+  background-color: #000;
+  border-radius: 0.25rem;
+  height: .5rem;
+  position: absolute;
+  transition-duration: .15s;
+  transition-property: transform;
+  transition-timing-function: ease;
+  width: 2.5rem;
+}
+.hamburger--slider .hamburger-inner {
+  top: 0.25rem;
+}
+
+.hamburger-inner, 
+.hamburger-inner:after, 
+.hamburger-inner:before {
+  background-color: #000;
+  border-radius: .5rem;
+  height: .5rem;
+  position: absolute;
+  transition-duration: .15s;
+  transition-property: transform;
+  transition-timing-function: ease;
+  width: 2.5rem;
+}
+.hamburger--slider .hamburger-inner:before {
+  top: .9rem;
+  transition-duration: .15s;
+  transition-property: transform,opacity;
+  transition-timing-function: ease;
+}
+
+.hamburger-inner, 
+.hamburger-inner:after, 
+.hamburger-inner:before {
+  background-color: #000;
+  border-radius: 0.25rem;
+  height: 0.25rem;
+  position: absolute;
+  transition-duration: .15s;
+  transition-property: transform;
+  transition-timing-function: ease;
+  width: 3.5rem;
+}
+.hamburger--slider .hamburger-inner:after {
+  top: 1.8rem;
+}
+.hamburger-inner:after {
+  bottom: -0.625rem;
+}
+
+.hamburger.is-active .hamburger-inner, 
+.hamburger.is-active .hamburger-inner:after, 
+.hamburger.is-active .hamburger-inner:before {
+  background-color: #000;
+}
+
+.hamburger--slider.is_active .hamburger-inner {
+  transform: translate3d(0,1.2rem,0) rotate(45deg);
+}
+
+.hamburger--slider.is_active .hamburger-inner:before {
+    opacity: 0;
+    transform: rotate(-45deg) translate3d(-0.6rem,-0.6rem,0);
```

```suggestion
    transform: rotate(-45deg) translate3d(-0.57152rem,-0.6rem,0);
```
I don't know why that weird number is there. In the original it's `-0.3752rem`, which when scaled up by 1.6 is `-0.57152rem` 🤷🏼‍♂️  Anyhow, I'm just copying it and making it exact.